src/waltz/http/fd_http_server.c

```diff
@@ -148,11 +157,18 @@ fd_http_server_new( void *                     shmem,
   uchar * _ws_recv_bytes  = FD_SCRATCH_ALLOC_APPEND( l,  1UL,                                          params.max_ws_recv_frame_len*params.max_ws_connection_cnt                            );
   struct fd_http_server_ws_frame * _ws_send_frames = FD_SCRATCH_ALLOC_APPEND( l, alignof(struct fd_http_server_ws_frame), params.max_ws_send_frame_cnt*params.max_ws_connection_cnt*sizeof(struct fd_http_server_ws_frame) );
   http->oring             = FD_SCRATCH_ALLOC_APPEND( l,  1UL,                                          params.outgoing_buffer_sz                                                            );
-
-  http->oring_sz  = params.outgoing_buffer_sz;
-  http->stage_err = 0;
-  http->stage_off = 0UL;
-  http->stage_len = 0UL;
+#if FD_HAS_ZSTD
+  uchar * _zstd_ctx       = FD_SCRATCH_ALLOC_APPEND( l,  16UL,                                         ZSTD_estimateCCtxSize( FD_HTTP_ZSTD_COMPRESSION_LEVEL )                              );
+  http->zstd_ctx = ZSTD_initStaticCCtx(_zstd_ctx, ZSTD_estimateCCtxSize( FD_HTTP_ZSTD_COMPRESSION_LEVEL ));
```

Comment:
NULL on error, and formatting, I'd also move the init stuff down below with the other joins()

```suggestion
  http->zstd_ctx = ZSTD_initStaticCCtx( _zstd_ctx, ZSTD_estimateCCtxSize( FD_HTTP_ZSTD_COMPRESSION_LEVEL ) );
```